### PR TITLE
Version 1.1.0 and CHANGELOG update

### PR DESCRIPTION
**Pull request by jreineckearm**
#### What it does

- Updates version to 1.0.0 for upcoming release.
- Updates `CHANGELOG.md` with changes since 1.0.1 release.

#### How to test

No code change.

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the contribution guidelines](https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/blob/main/CONTRIBUTING.md)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the code of conduct](https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
 
+## 1.1.0
+- Fixed variable fetch before variables are available.
+- Fixed variable decoration display if it starts before the loaded memory range.
+- Fixed `Memory: Show Memory Inspector` command to open a new Memory Inspector window instance instead of the `Output` channel.
+- Enabled reload of variables for variable decoration on successful `scopes` response of the Microsoft Debug Adapter Protocol (DAP).
+- Changed Memory Inspector `Output` channel to only show on first log message.
+- Changed usage of `Word` to `MAU (Minimum Addressable Unit)` to honor the different meaning of `Word` in different memory architectures.
+- Changed `Infinite` scrolling option to `Grow` to remove confusion about its meaning.
+- Changed used GUI framework library to [Primereact](https://primereact.org/).
```
Comment (thegecko) [inline CHANGELOG.md:11]:
```suggestion
- Changed GUI framework library to [Primereact](https://primereact.org/).
```

**Review by thegecko — APPROVED** [review]
One minor suggestion

**Comment by jreineckearm** [comment]
Thanks, @thegecko ! @colin-grant-work , @planger , are you equally happy? Someone with committer rights would need to merge.

**Review by planger — APPROVED** [review]
Thanks, looks good to me! :+1:

**Pull request merged**